src/pages/LeeSoo/Main/Feeds.js

```diff
@@ -0,0 +1,135 @@
+          ...commentList,
+          {
+            id: counter,
+            userName: 'Sooboi_97',
+            content: inputValues,
+            isLiked: false,
+          },
+        ]);
+        inputValuesChange('');
+      }
+    }
+  };
+  console.log(props);
+  return (
+    <div className="contents">
+      <article>
+        <div className="article-top">
+          <img
+            alt="유저 이미지"
+            src="/images/LeeSoo/userImg.jpg"
+            className="userImg"
+          />
+          <p>{props.title}</p>
+          <span className="dotMore">
+            <a href="#">
+              <i className="fa-solid fa-ellipsis" />
+            </a>
+          </span>
+        </div>
+        <div className="article-main">
+          <img alt="피드사진" src={props.source} />
+        </div>
+        <div className="article-bar">
+          <div className="bar-left">
+            <button>
+              <span className="bar-like">
+                <i className="fa-solid fa-heart fa-2x" />
+              </span>
+            </button>
+            <button>
+              <span className="bar-comments">
+                <i className="fa-solid fa-comment fa-2x" />
+              </span>
+            </button>
+            <button>
+              <span className="bar-share">
+                <i className="fa-solid fa-arrow-up-from-bracket fa-2x" />
+              </span>
+            </button>
+          </div>
+
+          <div className="bar-right">
+            <button>
+              <span className="bar-bookmark">
+                <i className="fa-solid fa-bookmark fa-2x" />
+              </span>
+            </button>
+          </div>
+          <div className="clearFloat" />
+        </div>
+        <div className="article-reaction">
+          <img alt="좋아요 유저파일" src="/images/LeeSoo/likeuser.jpg" />
+          <p>
+            <b>&nbsp;&nbsp;goophyishollyshakeit</b>님 외{' '}
+            <b>6명이 좋아합니다.</b>
+          </p>
+        </div>
+        <div className="Comments-show">
+          {/* 여기에 댓글 기능 구현하기 */}
+          <Comments list={commentList} />
+        </div>
+        {/* 여기에 댓글 입력창 구현하기 */}
+        <div className="Comments-write" onKeyDown={EnterCatch}>
+          <input
+            className="input-area"
+            type="text"
+            placeholder="내용을 입력하세요"
+            value={inputValues}
+            onChange={e => {
+              inputValuesChange(e.target.value);
+            }}
+          />
+
+          <button
+            className="input-bt"
+            onClick={() => {
+              {
+                setCounter(counter => counter + 1);
+                setCommentList([
+                  ...commentList,
+                  {
+                    id: counter,
+                    userName: 'Sooboi_97',
+                    content: inputValues,
+                    isLiked: false,
+                  },
+                ]);
+                inputValuesChange('');
+              }
+            }}
```

Comment:
콜백함수를 별개의 
```suggestion
const addComment=() => {
              {
                setCounter(counter => counter + 1);
                setCommentList([
                  ...commentList,
                  {
                    id: counter,
                    userName: 'Sooboi_97',
                    content: inputValues,
                    isLiked: false,
                  },
                ]);
                inputValuesChange('');
              }
            }
```  
별개의 함수로 만들어 이용해 주면   좋을 것 같아요!  혹시 콜백함수로 바로 사용하신 이유가 무엇일까용?